doc/architecture/script_command_interface.rst

```diff
@@ -0,0 +1,136 @@
+.. _script_command_interface:
+
+Script Command Interface
+========================
+
+The ``ScriptCommandInterface`` allows using functionality of the robot that would normally
+interrupt program execution. For this purpose a subset of the robot's functionality is made
+available in specific API calls.
+
+At the time of writing the ``ScriptCommandInterface`` provides the following functions:
+
+- ``zeroFTSensor()``: Zeros the force/torque sensor.
+- ``setPayload()``: Set the active payload mass and center of gravity.
+- ``setToolVoltage()``: Set the voltage of the tool output.
+- ``startForceMode()`` / ``endForceMode()``: Start and end a force mode. See the `force mode
+  example <https://github.com/UniversalRobots/Universal_Robots_Client_Library/blob/master/examples/force_mode_example.cpp>`_ for more information.
+- ``startToolContact()`` / ``endToolContact()``: Start and end a tool contact mode. See the `tool
```

Comment:
I think we a missing the force mode in this list

Reply:
I love the ```literalinclude``` includes. When we can ensure that the code we show in the documentation is valid, using our CI.

Reply:
> I think we a missing the force mode in this list

You mean apart from that?
```
- ``startForceMode()`` / ``endForceMode()``: Start and end a force mode. See the `force mode
  example <https://github.com/UniversalRobots/Universal_Robots_Client_Library/blob/master/examples/force_mode_example.cpp>`_ for more information.
```